src/completers/git.zsh

```diff
@@ -64,8 +64,18 @@ _fzf_complete_git() {
 
     local original_arguments=("${(Q)${(z)@}[@]}")
     local command_pos=${original_arguments[(i)$arguments[1]]}
+
     if (( $command_pos > 1 )); then
-        local -x ${${(Q)${(z)@}}[1, $command_pos - 1]}
+        local v
+        for v in "${${(z)@}[1, $command_pos - 1][@]}"; do
+            local name=${v%=*}
+            local cmd=${v#*=}
+            if [[ ${cmd[1]} = '~' ]]; then
+                local -x "$name=$(zsh -c "echo $cmd")"
+            else
+                local -x "$name=${(e)cmd}"
+            fi
+        done
```

Comment:
It seems to be working in most cases, but expansion by using [dynamic named directories](https://zsh.sourceforge.io/Doc/Release/Expansion.html#Dynamic-named-directories) which is defined in .zshrc isn't worked.

Reply:
As unexported shell variables are not inherited to a child zsh process, the following case does not look like being handled; I did not test this though.
```zsh
KUBECONFIG=~/.config/$CLUSTER kubectl get **<TAB>
```